### PR TITLE
Update chapter3

### PR DESCRIPTION
**Pull request by plusbang**
Update `optimize_model` usage in chapter3 and refactor structure.
(Both English and Chinese version)

### CODE REVIEW

File: ch_3_AppDev_Basic/3_Quick_Start.ipynb [hunk 1]
```diff
@@ -18,34 +18,43 @@
...
    "source": [
-    "from bigdl.llm.transformers import AutoModelForCausalLM\n",
+    "from transformers import LlamaForCausalLM, LlamaTokenizer\n",
+    "from bigdl.llm import optimize_model\n",
+    "\n",
+    "model_path = 'openlm-research/open_llama_3b_v2'\n",
     "\n",
-    "model = AutoModelForCausalLM.from_pretrained(pretrained_model_name_or_path=\"openlm-research/open_llama_3b_v2\",\n",
-    "                                             load_in_4bit=True)"
+    "tokenizer = LlamaTokenizer.from_pretrained(model_path)\n",
```
Comment (shane-huang) [inline ch_3_AppDev_Basic/3_Quick_Start.ipynb:51]: `tokenizer = LlamaTokenizer.from_pretrained(model_path)` put tokenizer initialization to Run LLM section

File: ch_3_AppDev_Basic/3_Quick_Start.ipynb [hunk 2]
```diff
@@ -71,23 +80,57 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "save_directory='./open-llama-3b-v2-bigdl-llm-INT4'\n",
+    "\n",
+    "model.save_low_bit(save_directory)"
+   ]
```
Comment (shane-huang) [inline ch_3_AppDev_Basic/3_Quick_Start.ipynb:86]: also show load usage. Refer to Notebook 4.1 save/load section.

Comment (plusbang) [reply ch_3_AppDev_Basic/3_Quick_Start.ipynb:89]: > also show load usage. Refer to Notebook 4.1 save/load section.

Have updated.

**Review by shane-huang — APPROVED** [review]
Merge for now. Will revise texts together.

**Pull request merged**